internal/routes/proxies/proxies.go

```diff
@@ -53,6 +61,9 @@ func CreateProxyHandler(targetURL string, path string, modifyRequest func(*http.
 func RegisterRoutes(r *gin.Engine, deps *Dependencies) {
 	r.Any("/api/v1/serve-proxy/:name/*path", handleServeProxy(deps))
 	r.Any("/api/v1/ray-dashboard-proxy/:name/*path", handleRayDashboardProxy(deps))
+	r.Any("/api/v1/auth/:path", handleAuthProxy(deps))
+	r.Any("/api/v1/:path", handlePostgrestProxy(deps))
+	r.Any("/api/v1/playgrounds/chat/:endpoint", handlePlaygroundProxy(deps))
```

Comment:
playground middleware is not need anymore, we can re-implement the client logic without this

Reply:
ok, remove it done